pandas/util/testing.py

```diff
@@ -585,7 +585,7 @@ def assert_index_equal(
     ----------
     left : Index
     right : Index
-    exact : bool / string {'equiv'}, default 'equiv'
+    exact : bool / str {'equiv'}, default 'equiv'
```

Comment:
I'd prefer `bool or {'equiv'}`, what do you think? (in all cases)

Reply:
@datapythonista yeah we could do that. I will make the required changes and resubmit it then